src/providers/tenderly-simulation-provider.ts

```diff
@@ -125,7 +125,7 @@ export class FallbackTenderlySimulator extends Simulator {
       ))
     ) {
       log.info(
-        'Simulating with eth_estimateGas since token is native or approved.'
+        'Simulating with eth_estimateGas since token is approved.'
```

Comment:
ooh wait can't we always do eth_estimateGas with the deposit address as from? i.e. no need for tenderly in this case

Reply:
like if you keep the conditional above and move the fromAddress update here this should work and save us tenderly costs +  latency

Reply:
ah yeah good idea